src/native-common/Image.tsx

```diff
@@ -179,10 +195,27 @@ export class Image extends React.Component<Types.ImageProps, Types.Stateless> im
         if (this.props.headers) {
             source.headers = this.props.headers;
         }
+        if (this._forceCache) {
+            source.cache = 'force-cache';
```

Comment:
Can you explain why you use force-cache instead of only-if-cached?

force-cache will try to make a network request if there's no data in the cache, which we already expect won't work (since we're in the failure path)